src/_playground/Routes.js

```diff
@@ -345,7 +354,8 @@ export class Routes extends Component {
                             {navItems}
                         </nav>
                     </div>
-                    <div className='frDocs-Content'>
+                    <div className={`frDocs-Content ${showNav}`}>
+                        <button className='fd-button sap-icon--menu2 fd-button--standard fd-button--light sidebar-toggle' onClick={this.toggleNavVis.bind(this)} />
```

Comment:
I think this button will need an `aria-label` and `aria-expanded` true/false toggle on it as well for a11y reasons.